lib/UUID.cpp

```diff
@@ -0,0 +1,33 @@
+/*
+ * UUID.cpp, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+
+#include "StdInc.h"
+#include "UUID.h"
+
+boost::uuids::uuid UUID::getID() const
+{
+	return ID;
+}
+
+boost::uuids::uuid UUID::generateID()
+{
+	static boost::uuids::random_generator_mt19937 gen;
```

Comment:
Build fails here. No such symbol `random_generator_mt19937`. Boost version problem?

Reply:
You're right, CI works with `boost 1.64` when `boost 1.67` provide for  convenience 
` random_generator_mt19937` that is definition of previous version  `basic_random_generator<mt19937>`